packages/pointer_interceptor/pointer_interceptor_ios/README.md

```diff
@@ -0,0 +1,15 @@
+# pointer_interceptor_ios
+
+A new Flutter plugin project.
```

Comment:
This should all be replaced; see other simple `*_ios` package READMEs (e.g., `url_launcher_ios`) for example text.